### PR TITLE
travis ci

### PR DESCRIPTION
**Pull request by sethladd**

### CODE REVIEW

**Comment by sethladd** [comment]
cc @devoncarew

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+language: node_js
```
Comment (devoncarew) [inline .travis.yml]: we may want to change this to `dart` later, in order to validate the dart code that's generating the metrics

**Comment by devoncarew** [comment]
lgtm!

File: .travis.yml (hunk 1 above)
Comment (sethladd) [inline .travis.yml]: cool, as long as we can install firebase via npm later.

Comment (devoncarew) [inline .travis.yml]: I _think_ every image has npm available: https://docs.travis-ci.com/user/ci-environment/#Runtimes

**Pull request merged**